### PR TITLE
Adding POST Requests

### PR DESCRIPTION
**Pull request by vivek-wednesday**
## Description

- [X]  Fixed issues on PR: vivek-wednesday/cab-booking-nodejs#1
- [X]  Added POST request methods for bookings, cab and drivers
- [X]  Added test for POST requests Daos
- [X]  Auth for POST requests
- [X]  Test for POST requests route

### 1. Add cabs
ezgif com-gif-maker (5)
### 2. Create bookings
ezgif com-gif-maker (6)

### CODE REVIEW

**Comment by vivek-wednesday** [comment]
## Auth for POST request
Screenshot 2021-09-24 at 3 30 56 PM

**Comment by vivek-wednesday** [comment]
## Tests coverage
Screenshot 2021-09-24 at 3 32 51 PM
Screenshot 2021-09-24 at 3 32 25 PM

**Comment by vivek-wednesday** [comment]
## Scope for requests
ezgif com-gif-maker (7)

File: lib/routes/bookings/user/tests/routes.test.js [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
-import { mockData } from 'utils/mockData';
+import { mockData, createMockTokenWithScope } from 'utils/mockData';
+import { SCOPE_TYPE, OAUTH_CLIENT_ID } from 'utils/constants';
 
 const { MOCK_BOOKING: bookings } = mockData;
 
-describe('/user route tests ', () => {
+describe('/user route tests', () => {
     let server;
+    const superAdminToken = createMockTokenWithScope(
```
Comment (christin-wednesday) [inline lib/routes/bookings/user/tests/routes.test.js:9]: We could add tests to check if invalid scopes are rejected

Comment (vivek-wednesday) [reply lib/routes/bookings/user/tests/routes.test.js:9]: Hmm interesting, we could do that.

**Pull request merged**